Formula/b/bfs.rb

```diff
@@ -30,8 +30,10 @@ class Bfs < Formula
   def install
     ENV.llvm_clang if OS.mac? && DevelopmentTools.clang_build_version <= 1300
 
-    system "make", "release"
-    system "make", "install", "PREFIX=#{prefix}"
+    system "./configure", "--enable-release"
+    system "make"
+    system "make", "install", "DEST_PREFIX=#{prefix}", "DEST_MANDIR=#{man}"
```

Comment:
You should be able to do `"./configure", ..., "--prefix=#{prefix}", "MANDIR=#{man}"`